src/domain/app/i18n/en.json

```diff
@@ -1669,7 +1686,8 @@
       "parentOrganizationDesc": "Parent organization, descending"
     },
     "title": "Organizations",
-    "warningNoRightsToEdit": "You have insufficient rights to edit this organization."
+    "warningNoRightsToEdit": "You have insufficient rights to edit this organization.",
+    "warningNoRightsToEditMerchant": "Only superusers can manage web store merchants"
```

Comment:
Financial admins should also be mentioned here as they too should be able to manage merchants.

Reply:
This is fixed in the most recent commit